composer.json

```diff
@@ -1,31 +1,30 @@
 {
     "name": "heyday/silverstripe-cacheinclude",
-    "type": "silverstripe-module",
+    "type": "silverstripe-vendormodule",
     "require": {
-        "php": ">=5.3.2",
-        "composer/installers": "~1.0",
-        "doctrine/cache": "~1.3",
-        "symfony/yaml": "~2.4",
-        "symfony/expression-language": "~2.4",
-        "psr/log": "~1.0"
+        "silverstripe/framework": "^4",
```

Comment:
Note to self: this needs to be changed to `^4.3`. Earlier versions don’t provide automatic cache segmentation for draft/live